src/blocks/content.scss

```diff
@@ -0,0 +1,7 @@
+.content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  padding: 30px;
+}
```

Comment:
You can move to the variables file as well. It's used only once, but it may get difficult to find it, if you want to change it in the future, what do you think?